components/overflow-group/README.md

```diff
@@ -1,10 +1,40 @@
 # Overflow Groups
+The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. 
 
-## d2l-overflow-group
-The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. Items added to this container element will no longer wrap onto a second line when the container becomes to small but will be added too a dropdown menu with configurable styling.
+<!-- docs: demo size:medium autoSize:false display:block -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/overflow-group/overflow-group.js';
```

Comment:
Would this need the other imports? Like dropdown button etc.

Reply:
The buttons/dropdowns that get used are imported within the component so we don't technically need to add them here.